Tea

```diff
@@ -0,0 +1,6 @@
+# https://tea.xyz/what-is-this-file
+---
+version: 1.0.0
+codeOwners:
+  - '0x5693a0ad9a6E3f92d1243FF56A5E4b1f162785fa'
```

Comment:
I don't know much about the tea protocol.  Will this token make you the owner of the project or does it include your token to be remunerated for future contributions to this project?